consensus/service/src/peer_keepalive.rs

```diff
@@ -141,7 +141,11 @@ impl PeerKeepalive {
 
                     match conn_manager
                         .conn(&responder_id)
-                        .ok_or_else(|| RetryError::Internal(format!("{} not found", responder_id)))
+                        .ok_or_else(|| RetryError {
+                            error: mc_peers::Error::NotFound,
```

Comment:
I'm up for suggestions if there is a better error to be returning here.
The logging down in the Err() handling below should provide the responder id and a message, so didn't feel this was losing info